ballerina/websocket_service.bal

```diff
@@ -25,68 +24,73 @@ isolated service class WsService {
     private final Engine engine;
     private final readonly & __Schema schema;
     private final Context context;
-    private map<[websocket:Caller, parser:OperationNode]> clientsMap;
-    
+
     isolated function init(Engine engine, __Schema schema) {
         self.engine = engine;
         self.schema = schema.cloneReadOnly();
         self.context = new();
-        self.clientsMap = {};
-    }
-                
-    isolated remote function onClose(websocket:Caller caller, int statusCode, string reason) {
-        lock {
-            _ = self.clientsMap.remove(caller.getConnectionId());
-        }
     }
 
     isolated remote function onTextMessage(websocket:Caller caller, string data) returns websocket:Error? {
         lock {
```

Comment:
I think we can remove this now. This was added due to the `client map` that we have in the earlier design. Shall we try this without `lock`?